go/teams/create.go

```diff
@@ -29,29 +29,17 @@ func CreateImplicitTeam(ctx context.Context, g *libkb.GlobalContext, impTeam key
 	}
 
 	// Load all the Keybase users
-	loadUpakByUsername := func(username string) (*keybase1.UserPlusKeysV2, error) {
-		uid, err := g.GetUPAKLoader().LookupUID(ctx, libkb.NewNormalizedUsername(username))
-		if err != nil {
-			g.Log.CDebugf(ctx, "CreateImplicitTeam: failed to load uid: %s msg: %s", username, err)
-			return nil, err
-		}
-		upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{
-			UID: uid,
-		})
-		if err != nil {
-			g.Log.CDebugf(ctx, "CreateImplicitTeam: failed to load user: %s msg: %s", username, err)
-			return nil, err
-		}
-		return &upak.Current, nil
-	}
-
 	loadUsernameList := func(usernames []string) (res []*keybase1.UserPlusKeysV2, err error) {
 		for _, username := range usernames {
-			upak, err := loadUpakByUsername(username)
+			upak, _, err := g.GetUPAKLoader().LoadV2(libkb.LoadUserArg{
+				Name:       username,
```

Comment:
I'm not sure what you mean. This is CORE-5873 "give UPAK loader ability to load via username", and I assumed by the timing when you made that ticket that this piece of code was when you decided that that would be a nice feature.